src/common/base/NebulaKeyUtils.cpp

```diff
@@ -8,6 +8,8 @@
 
 namespace nebula {
 
+const std::string kIndexPrefix = "_i_";           // NOLINT
```

Comment:
Do we still need it?  The first byte could be used as the prefix.

Reply:
> Do we still need it? The first byte could be used as the prefix.

make sense !  Will using kIndex for prefix.